Gems/Terrain/Code/Source/Components/TerrainLayerSpawnerComponent.h

```diff
@@ -33,9 +33,10 @@ namespace Terrain
     {
         static const AZ::u32 s_backgroundLayer = 0;
         static const AZ::u32 s_foregroundLayer = 1;
-        static const AZ::u32 s_priorityMin = 0;
-        static const AZ::u32 s_priorityMax = 10000; //arbitrary number because std::numeric_limits<AZ::u32>::max() always dislays -1 in RPE
-        static const AZ::u32 s_prioritySoftMax = 100; //design specified slider range
+        static const int32_t s_priorityMin = std::numeric_limits<int32_t>::min();
+        static const int32_t s_priorityMax = std::numeric_limits<int32_t>::max();
```

Comment:
I'm going to change this back to a smaller range (probably -10000 to 10000) based on discussions with @mbalfour-amzn about layers and possibly merging priority under the hood.